hooks/post-command

```diff
@@ -59,6 +51,21 @@ docker_metadata_file_exists() {
 # Hook
 ######
 
+[[ -z "${TRIVY_EXE_PATH}" ]] \
+  && fail_with_message "TRIVY_EXE_PATH env. is unset or empty"
+
+[[ ! -f "${TRIVY_EXE_PATH}" ]] \
+  && fail_with_message "trivy exe file does not exist ('${TRIVY_EXE_PATH}')"
+
+# I would prefer to just execute the path pointed-to by this environment
+# variable - but that is incompatible with the testing framework. As a
+# result, we need to modify PATH to include the directory containing
+# the trivy executable.
```

Comment:
wait, so we can't just execute `"${TRIVY_EXE_PATH}"` instead? so we wouldn't need to rely on `PATH`

Reply:
We cannot use the terrible `stub` stuff if we use the file path :(